vocabularies/georesources-report-types.ttl

```diff
@@ -362,7 +362,7 @@ grt:petroleum-end-authority a skos:Concept ;
     skos:altLabel "End of Authority Report (DAA)"@en,
         "End of Authority Report (PSL)"@en,
 	"Petroleum Report - End of Authority Report"@en;
-    skos:definition "A report that is required to be lodged at the end of tenure over a Data Acquisition Authority, detailing the work undertaken on the permit throughout the life of tenure."@en  ;
+    skos:definition "A report that is required to be lodged at the end of tenure over a Data Acquisition Authority, detailing the work undertaken on the authority throughout the life of tenure."@en  ;

```

Comment:
Should this also cover Petroleum Survey License (PSL)?

Reply:
```suggestion
    skos:definition "A report that is required to be lodged at the end of tenure over a Data Acquisition Authority or Petroleum Survey License, detailing the work undertaken on the authority throughout the life of tenure."@en  ;

```

Good pick up.